backend/src/main/java/org/eclipse/tractusx/puris/backend/delivery/controller/DeliveryRequestApiController.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2023, 2024 Volkswagen AG
```

Comment:
Why 2023, 2024? If code is pretty close to item stock (nearly identic) ok, else 2024 only